CHANGELOG.rst

```diff
@@ -1,5 +1,33 @@
 Changelog
 =========
+1.2 (2019-08-09)
+----------------
+MLflow 1.2 includes the following major features and improvements:
+
+- Experiments now have editable tags and descriptions (#1630, #1632, #1678, @ankitmathur-db)
+- Search latency has been significantly reduced in the SQLAlchemyStore (#1660, @t-henri)
+
+**More features and improvements**
+
+- [Tracking] Backend stores now support run tag values up to 5000 characters in length. Some store implementations may support longer tag values (#1687, @ankitmathur-db)
+- [Scoring] Gunicorn options can now be configured for the ``mlflow models serve`` CLI with the ``GUNICORN_CMD_ARGS`` environment variable (#1557, @LarsDu)
+- [UI] Jsonnet artifacts can now be previewed in the UI (#1683, @ankitmathur-db)
```

Comment:
Do you need [Tracking], [Scoring], etc in front of these? It looks kind of weird.

Also, some people might not know what gunicorn is so maybe call it "gunicorn server options" or something like that.

Reply:
We've historically added the component tags ([Tracking], [Scoring], etc), they seem helpful for quickly skimming & identifying changes to components you're interested in (although maybe we should reformat our changelog to list changes by component), so I kept them in while addressing your comments - updated "gunicorn options" -> "gunicorn server options" though

Reply:
I think I always commented to remove them in settings like this! It might make sense to keep them for the bug fix list but this list is too small to benefit from them here.. they just make it harder to read.

Reply:
Ah got it, yeah I'll remove them from this list :)